### PR TITLE
refactor for agents

### PR DESCRIPTION
**Pull request by ChBernat**
Refactored front-end's agent module.

### CODE REVIEW

File: console/src/app/agents/models/group.model.ts [hunk 1]
```diff
@@ -1,37 +1,37 @@
-import { Agent } from './agent.model';
+import {Agent} from '../../services/agent/agent.model';
```
Comment (theflexbird) [inline console/src/app/agents/models/group.model.ts]: Why you here did not use a path like `@app`?

File: server/package.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
   "main": "app.js",
   "scripts": {
     "spec": "speculate --name=kaholo-server",
-    "test": "jest api/tests -w 1",
+    "test": "jest api/tests -w 1 --forceExit",
```
Comment (theflexbird) [inline server/package.json]: Why you added ` --forceExit`

Comment (ChBernat) [reply server/package.json]: Tests did not want to finish without it and so I could not push :(

Comment (theflexbird) [reply server/package.json]: I know what it is doing. Why ment "Do you know the cause of why it is not exiting?".

**Pull request merged**